File: fizzbuzz.py
#!/usr/bin/python2.7

for i in range(100):
    i = i+1;
    out = ""
    
    if (i % 3 == 0):
        out += "Fizz"
    if (i % 5 == 0):
        out += "Buzz"
    if (len(out) == 0):
        out = str(i)

    print out

File: fizzbuzz.rb
#!/usr/bin/ruby1.9.1

fizzbuzzes = Range.new(1,100).map do |v|
    r = ""
    r += "Fizz" if v % 3 == 0
    r += "Buzz" if v % 5 == 0
    r = v if r.length == 0
    r
end

puts fizzbuzzes
